src/components/Content/ContentSettings.vue

```diff
@@ -80,6 +80,19 @@
                 for="showVirtualLevels"
               >Show Virtual Levels Beyond {{maxLevel}}</label>
             </div>
+            
+            <div class="custom-control custom-switch">
+              <input
+                v-model="showFullValues"
+                type="checkbox"
+                class="custom-control-input"
+                id="showFullValues"
+              />
+              <label
+                class="custom-control-label"
+                for="showVirtualLevels"
+              >Show Full Values for Item Counts (can look ugly with huge numbers)</label>

```

Comment:
```suggestion
              >Display Full Item Counts in Jobs (may look ugly with huge numbers)</label>
```
This doesn't change how item counts are displayed elsewhere (like the money in the top left) so specifying that it's for jobs may help with some confusion.